src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -43,6 +44,7 @@ impl QemuInner {
             qemu_process: None,
             config: Default::default(),
             devices: Vec::new(),
+            netns: None,
```

Comment:
As discussed on #runtime-rs, this commit should also assign to `QemuInner::netns` in `prepare_vm()`, otherwise this won't work.

Reply:
Thx @pmores I re-checked my draft code in my local repo and I find that `self.netns = netns` indeed exists. 
I accidentally missed this when pushing the code. I will fix it.